context/discovery-ontology-chowlk.xml

```diff
@@ -1,58 +1,2 @@
-          <mxGeometry x="800" y="560" width="160" height="30" as="geometry" />
-        </mxCell>
-        <mxCell id="13" value="&lt;div&gt;&lt;b&gt;dc:creator:&lt;/b&gt;&amp;nbsp;Andrea Cimmino&amp;nbsp;&lt;/div&gt;&lt;div&gt;&lt;b&gt;dc:creator:&lt;/b&gt; Farshid Tavakolizadeh&lt;/div&gt;&lt;div&gt;&lt;b&gt;owl:versionInfo:&lt;/b&gt; 0.1.0&lt;/div&gt;&lt;div&gt;&lt;b&gt;dc:title:&lt;/b&gt;&amp;nbsp;WoT discovery ontology&lt;/div&gt;" style="shape=document;whiteSpace=wrap;html=1;boundedLbl=1;labelBackgroundColor=#ffffff;strokeColor=#000000;fontSize=12;fontColor=#000000;size=0.23333333333333334;" vertex="1" parent="1">
-          <mxGeometry x="120" y="360" width="250" height="90" as="geometry" />
-        </mxCell>
-      </root>
-    </mxGraphModel>
-  </diagram>
-</mxfile>
+<mxfile host="app.diagrams.net" modified="2021-04-26T09:36:02.763Z" agent="5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.85 Safari/537.36" version="14.6.6" etag="XIRbDmwrvtHAUBXOKZq0"><diagram id="LhQrBxsUQyT8V3tOavLX">7Vrfb6M4EP5rIt09NMIQSHhM0/bupF3t6bbS7j464IAVwJxxmrR//Y3B/LAhaa5NdvtAFCXMYI899nyfx4aJs0oPf3Ccx59ZSJKJbYWHiXM3sW00sz34k5rnSuPZVqWIOA1VoVbxlb4QpayL7WhICq2gYCwRNNeVAcsyEghNhzlne73YhiV6qzmOSE/xNcBJX/uNhiKutAt73ur/JDSK65aR51d3UlwXVp4UMQ7ZvqNy7ifOijMmqqv0sCKJHLx6XKp6D0fuNh3jJBPnVLCrCk842SnfJraXQNXbkD7BZVRdFgF7Ihxmawk6nOagzNaF/IuFkIO+lE3YDxEV8W49DVgKwt4J5C8TN60B26ntr3ltXoSv2t3v99O9M2U8AsG2kF8Zhl+oPGSz1kCJjiMDvhVBTFJcNmTJNpsmqxuqyWPmyikUz3VcwGzm8jJjAv5u9zEV5GuOA6nbAxRKv9IEJCR7i4NtxNkuC7/sREIzovQh5tsvUIsKCQ9rarlNWzCGghyOzjVqIgigR1hKhBxyq65Qw0ehbjZX8r6NYdtXurgbv5ZSYoWbqLHdhhZcqOgajjSnF2nlvD/GNIt6A1mOCpE1rdfHschw/sj+ZrQcBqnJ5XXZV/cWvtYUBmZlTVwwvSpl25AdQ54ZsmvIniHPDXlhyL4uSwn6pMuOIbuGPDdkv2Nf+ocM/5DhHzL8Q4Z/yPAPGf4hwz9k+NfKyPAPGf4hwz9k+Ida/+B7kdifzV099r1+7CPL7ce+c4HQn/VCv8uo/5CIFoJjQVn2V7ZhPC0vR0yMmLguJhaWvh641gAmhtaDS2DCHUg8jIAnWbiUyRpIQYKLggZ6jJMDFd/LFXKhpB+dO3cHhZRSeK6FDPr5vSv8KC24tdhWK6W2Xp2GyrYwD5Qk7wF42bZJBMv2Q0gXlSOMi5hFLMPJfatV0yfLaZNXsB0PiLZgCswjIjQiOWOKO1PoDsxgreMkAbJ50jsxNK2qhYpQWladGRFkG5FR+aNqddNQw5DnzE8bqkahZ6iMssbtswLPO5rxFjnOtAj0/t0xoSVqNwFLGK+yRR6tf7NngHpJPfBrz/z22rV+b+v38+g2nawaVfmks4xxcXQ9+Em9PJlMa901AQtUJHSI4oRGmcQvxCuBHt1KwqKwkVqqGykNwxIQnBT0Ba+TGlT6ciUZL8Frktw2Tq4qH6FdZ1N+TpGi2ggq++32q4sk9yRZ3gB52/U27p2wMYObbTYFeW9cz/8Xoa4TFmz1udqwTHQG1YKP72s063ZotmHWIzQbPlDZVYN0W55tqPpjka7fJ13ng5HujbGNcz33baRrGpr5hqHLke7iZAb8iWbbO1IEnOZj5jtmvtfPfF0jb2nOH7uZ7xCEL5H5+iexcEc5LFQMWHAExAiInwUI5xxALK50PFIvQ2Pq8q7UZfHxUxfHmU/1czjPfeOO0e2Zcs1T6sulLwj1IrR5bhBiQVacwG848vTI0yd4OsRFXAYDutL5nTNA2tc6v0P9J4caJD6zkG7oiIkRE78WE0PPea6Gif4zzoEnzs35WnMsGABmArmEqEO75rRt3amjPR1fZiGUh6ZWNE1pxsz7Zz/+fltnrAfMixjctq1H/IS3LKEvOCTx25tlexjUJUx9oU49j7cN4SdB8S4HBRWSj84a62/sEVrtnN1akG1Cshk9H+uEwXj1WwEhC3ZpGcevvhlQceSnda044+yzyiqNLNiyVHas0k9g7cFsuSxXqBR1Cnma9pldBpvI1rHpDGBzMPv0L4HN/kPYZr0ihxx23cW4VI1L1a9cqvzrLVUgtu+UVRug9s085/4/</diagram></mxfile>
```

Comment:
Not sure what the heck this is, but I don't think it belongs here.  Also there are embedded quotes and angle brackets in this thing which may cause parsing issues.

Reply:
never mind the syntax issue, still would rather see an SVG here or something standard.  But can live with it for now.  Not sure it is even used?   Is it just for bookkeeping by the tool?

Reply:
@AndreaCimminoArriaga any idea?

Reply:
Hi, is the file that generates the tool that I used to generate the ontology diagram. It is like the plantUML files.